src/main/webapp/filesFragment.xhtml

```diff
@@ -288,37 +287,36 @@
                         <span class="glyphicon glyphicon-pencil"/> #{bundle['file.editFiles']} <span class="caret"></span>
                     </button>
                     <ul class="dropdown-menu multi-level pull-right text-left" role="menu">
-                        <li class="#{DatasetPage.lockedFromEdits ? 'disabled' : ''}">
```

Comment:
I apologize for all this mess. I tried to use a div to replace repeated calls in the <li> tags. It didn't work right and because my page caching fixes solve this anyways I undid it, but it looks wild in github.